include/swift/Runtime/Config.h

```diff
@@ -131,6 +131,8 @@
 #define SWIFT_CC_DefaultCC_IMPL SWIFT_CC_c
 #define SWIFT_LLVM_CC_DefaultCC llvm::CallingConv::C
 
+#define SWIFT_CC_SwiftCC SWIFT_CC_swift
```

Comment:
The Swift calling convention is called `SWIFT_CC_swift` directly and `SWIFT_LLVM_CC_SwiftCC` to refer to the LLVM name. These need to match (either `swift` or `SwiftCC`) for the purposes of the macros in [RuntimeFunctions.def](include/swift/Runtime/RuntimeFunctions.def), so I chose to alias `SwiftCC` to `swift` here.

Reply:
I don't understand why this is necessary. RuntimeFunctions.def already uses `SwiftCC` for two functions. Why is this only now a problem?

Reply:
Previously, both `SwiftCC` functions in RuntimeFunctions.def used the `FUNCTION` macro. However, `swift_allocBox` uses `FUNCTION_WITH_GLOBAL_SYMBOL_AND_IMPL`.

In `RuntimeEntrySymbols.cpp`, that `FUNCTION_WITH_GLOBAL_SYMBOL_AND_IMPL` expands into `DEFINE_SYMBOL`, whereas `FUNCTION`, as defined in that same file, expands to nothing for any calling convention other than `RegisterPreservingCC`.

`DEFINE_SYMBOL` uses the `SWIFT_CC` macro, which expands to `SWIFT_CC_##CC`. Since SWIFT_CC_SwiftCC was undefined, I defined it here.

Reply:
Got it. That's fine.